Makefile

```diff
@@ -8,8 +8,7 @@ export TERRAFORM_VERSION := 1.7.5
 
 export TERRAFORM_PROVIDER_SOURCE := grafana/grafana
 export TERRAFORM_PROVIDER_REPO := https://github.com/grafana/terraform-provider-grafana
-# UPGRADE THE go.mod also!
-export TERRAFORM_PROVIDER_VERSION := 3.13.1
+export TERRAFORM_PROVIDER_VERSION := $(shell grep terraform-provider-grafana go.mod | awk '{print $$2}' | sed 's/v//')
```

Comment:
Validated the output as:

```
 grep terraform-provider-grafana go.mod | awk '{print $$2}' | sed 's/v//'
        github.com/grafana/terraform-proider-grafana/v3 v3.15.0
```

Reply:
In the local shell you have to remove the additional `$` in the awk statement, the double `$$` is an escape mechanism for Makefile.